src/main/java/uk/gov/companieshouse/itemhandler/ItemHandlerApplication.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.itemhandler;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class ItemHandlerApplication {
+
+	public static final String APPLICATION_NAMESPACE = "orders.api.ch.gov.uk";
+
```

Comment:
I think this namespace should reflect the actual application - item handler in this instance.

Reply:
updated